tracer/build/_build/Build.Utilities.cs

```diff
@@ -314,6 +314,30 @@ partial class Build
         .Description("Updates verified snapshots files with received ones")
         .Executes(ReplaceReceivedFilesInSnapshots);
 
+    Target CleanSnapshots => _ => _
+      .DependentFor(
+           RunOsxIntegrationTests, BuildAndRunOsxIntegrationTests,
+           RunLinuxIntegrationTests, BuildAndRunLinuxIntegrationTests,
+           RunWindowsIntegrationTests, BuildAndRunWindowsIntegrationTests,
+           RunDebuggerIntegrationTests, BuildAndRunDebuggerIntegrationTests,
+           RunWindowsAzureFunctionsTests, BuildAndRunWindowsAzureFunctionsTests,
+           RunWindowsTracerIisIntegrationTests,
+           RunWindowsSecurityIisIntegrationTests,
+           RunWindowsMsiIntegrationTests
+       )
+      .Description("Cleans snapshots produced by previous test runs")
+      .Executes(() =>
+       {
+           var snapshotsDirectory = TestsDirectory / "snapshots";
```

Comment:
FYI, there are snapshot directories in other location too - e.g. in Datadog.Trace.Tests 😄

Reply:
I based this on `PrintSnapshotsDiff`, the goal really is to not print a diff when it's not related to the current run